stable/appmesh-controller/Chart.yaml

```diff
@@ -1,6 +1,19 @@
-apiVersion: v2
+apiVersion: v1
 name: appmesh-controller
-description: The AWS App Mesh controller Helm chart for Kubernetes
-type: application
+description: App Mesh controller Helm chart for Amazon EKS
```

Comment:
is there actually anything specific to EKS contained in these charts? AFAICTR, the previous description was more appropriate since we want to serve all k8s users on AWS, not just EKS users.